src/lib/mongoBackend/TriggeredSubscription.cpp

```diff
@@ -35,13 +35,13 @@
 */
 TriggeredSubscription::TriggeredSubscription
 (
-  long long          _throttling,
-  long long          _lastNotification,
-  Format             _format,
-  const std::string& _reference,
-  AttributeList      _attrL,
-  const std::string& _cacheSubId,
-  const std::string& _tenant
+  long long           _throttling,
+  long long           _lastNotification,
+  Format              _format,
+  const std::string&  _reference,
+  AttributeList       _attrL,
+  const std::string&  _cacheSubId,
+  const char*         _tenant
```

Comment:
Taking into account that the other paramters (_reference and _cacheSubId) are strings, shouldn't be this also a string? Or maybe there is a strong reason to change it to char*...